src/components/molecules/AddNewChatModal/AddNewChatModal.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import Button from "@mui/material/Button";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+import Dialog from "@mui/material/Dialog";
+import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+import DialogContentText from "@mui/material/DialogContentText";
+import DialogTitle from "@mui/material/DialogTitle";
+import ReactDom from "react-dom";
+import { Autocomplete, Chip } from "@mui/material";
+import { Logo } from "../../atoms";
+import styles from "./AddNewChatModal.module.css";
+
+type ModalProps = {
+  open: boolean;
```

Comment:
```suggestion
  isOpen: boolean;
```
Samo "open" sugeruje, że to jest funkcja, którą gdy wywołamy, to otworzy nam coś.